src/sounds.cpp

```diff
@@ -21,7 +21,7 @@
 #ifdef SDL_SOUND
 #include "SDL2/SDL_mixer.h"
 #include <thread>
-#if (defined _WIN32 || defined WINDOWS)
+#if (!defined __MINGW64__ && defined _WIN32 || defined WINDOWS )
```

Comment:
Yes you are right. Come to think of it is WINDOWS really a macro?. it should be _WINDOWS even than I don't know if that it necessary..